### PR TITLE
Merge beta 19.7.0.2 into `trunk`

### PR DESCRIPTION
**Pull request by mokagio**
Includes:

- [Fix Site Name screen layout on smaller iPhones](https://github.com/wordpress-mobile/WordPress-iOS/pull/18419) by @Gio2018 
- [Mobile Gutenberg 1.74.1](https://github.com/wordpress-mobile/WordPress-iOS/pull/18445) by @geriux  – Notice that this is an Android-only hotfix, which was integrated in here for consistency only
- [A non-user facing release tooling update](https://github.com/wordpress-mobile/WordPress-iOS/pull/18439) by @AliSoftware 

---

Got a conflicts in `WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameView.swift` because:

- https://github.com/wordpress-mobile/WordPress-iOS/commit/0caf93774e93aef990f751e4ffd6abf972d5b119 changed it on `trunk`
- https://github.com/wordpress-mobile/WordPress-iOS/commit/3be617f694432a2df00a450fb18ab1439a93e08c changed in on `release/19.7`

The commit on `trunk` was older (by one day) but the code it results with seems better to me because it doesn't use arbitrary emojis as delimiters for the selected site name.

I ensured the app built, but did not run the tests locally or manually verified the how the screen where `SiteNameView` is used renders.

The conflict was:

```diff
diff --cc WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameView.swift
index 28af882df1,85375f6e2b..0000000000
--- a/WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameView.swift
+++ b/WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameView.swift
@@@ -144,15 -142,11 +144,23 @@@ private extension SiteNameView

      /// Highlghts the site name in blue
      func setupTitleColors() {
++<<<<<<< HEAD
 +        // enclose the vertical name between two characters that are not in the title
 +        // (and reasonably never will..) to distinguish it from any substring in the title
 +        let selectedVerticalName = "😎" + siteVerticalName + "🙃"
 +        let fullTitle = String(format: TextContent.title, selectedVerticalName)
 +        var attributedTitle = NSMutableAttributedString(string: fullTitle)
 +        // Use default title if the vertical name is empty or too long
 +        guard let range = fullTitle.nsRange(of: selectedVerticalName),
 +              !siteVerticalName.isEmpty,
 +              siteVerticalName.count <= Metrics.verticalNameDisplayLimit else {
++=======
+         // find the index where the vertical name goes, so that it won't be confused
+         // with any word in the title
+         let replacementIndex = NSString(string: TextContent.title).range(of: "%@")
+
+         guard !siteVerticalName.isEmpty, replacementIndex.length > 0 else {
++>>>>>>> origin/trunk
              titleLabel.setText(TextContent.defaultTitle)
              return
          }
```

@Gio2018 can you take a look at my conflicts resolution and validate it was correct? Thanks!

---

**Update:** While waiting for feedback, this PR got conflicts with `trunk`. [`0fe4b03`](https://github.com/wordpress-mobile/WordPress-iOS/pull/18463/commits/0fe4b03938c7d61cf8dce468924a43ead5cd8a72) resolves them. They were the usual Gutenberg hotfix vs stable conflicts. Please refer to the commit message for more details.

### CODE REVIEW

File: WordPress/Resources/ar.lproj/Localizable.strings [hunk 1]
```diff
@@ -9497,6 +9494,9 @@ translators: %s: Select control option value e.g: \"Auto, 25%\". */
 /* Sentence to justify why the app asks permission from the user to access their Media Library. */
 "infoplist.NSPhotoLibraryUsageDescription" = "لإضافة صور أو مقاطع فيديو إلى مقالاتك.";
 
+/* Footer text for Invite Links section of the Invite People screen. */
+"invite_people_invite_link_footer" = "استخدم هذا الرابط لضم أعضاء فريقك من دون دعوتهم واحدًا تلو الآخر. سيتمكن أي شخص يزور عنوان URL هذا من التسجيل في مؤسستك، حتى إذا تلقى الرابط من شخص آخر، لذا تأكَّد من مشاركته مع أشخاص موثوق بهم.";
```
Comment (mokagio) [inline WordPress/Resources/ar.lproj/Localizable.strings:9498]: This pair of removed and added key is a result of the change from [this commit](https://github.com/wordpress-mobile/WordPress-iOS/pull/18425/commits/8ac6584469211cf5206b853b752c5afc119b3f84). Same for all the other locales.

File: WordPress/Resources/zh-Hans.lproj/Localizable.strings [hunk 2]
```diff
@@ -7768,7 +7826,7 @@ translators: %s: Select control option value e.g: \"Auto, 25%\". */
 "To use stats on your site, you'll need to install the Jetpack plugin." = "要在您的网站上使用“统计”功能，您需要安装Jetpack插件。";
 
 /* Message explaining that Jetpack needs to be installed for a particular site. Reads like 'To use this app for example.com you'll need to have... */
-"To use this app for %@ you'll need to have the Jetpack plugin installed and activated." = "要将此应用用于 ％@，您需要安装并启用 Jetpack 插件。";
+"To use this app for %@ you'll need to have the Jetpack plugin installed and activated." = "要将此应用用于 %@，您需要安装并启用 Jetpack 插件。";
```
Comment (mokagio) [inline WordPress/Resources/zh-Hans.lproj/Localizable.strings:7829]: The previous version had the wrong `%` character which wouldn't have worked in `String(format:)`.

**Review by mokagio — COMMENTED** [review]
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [x] Diffs from the PRs that made it into this beta

File: WordPress/Resources/zh-Hans.lproj/Localizable.strings (hunk 2 above)
Comment (AliSoftware) [reply WordPress/Resources/zh-Hans.lproj/Localizable.strings:7829]: Properly flagged by the linter, I presume?

**Review by AliSoftware — COMMENTED** [review]
On the release management side of things (`.strings` updates, version bump in `xcconfig` etc) all looks good 👍 

But I'll leave the honor of approving (or not) to @Gio2018  so he can double check and validate the conflict resolution is correct

**Comment by twstokes** [comment]
👋 @mokagio your conflict resolution looks good to me as it [matches the comment here](https://github.com/wordpress-mobile/WordPress-iOS/pull/18410#discussion_r854501678) that initiated it. 👍

**Comment by mokagio** [comment]
I'm going to consider @twstokes and @AliSoftware 's comments a combined approval and ~~admin-merge this PR~~ ask for someone else to approve because I can't admin-merge onto `trunk`.

**Review by oguzkocer — APPROVED** [review]

**Pull request merged**